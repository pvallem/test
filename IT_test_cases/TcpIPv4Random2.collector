bidir_flow_info {
  key {
    src_address: "\"Y\275\211"
    dst_address: "\256\027\261\263"
    src_port: 28189
    dst_port: 15208
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201243430
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563201243430
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 170
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563201243430
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "Z\315\226a"
    dst_address: "_\037\222x"
    src_port: 25617
    dst_port: 35224
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201077906
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563201077906
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 150
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563201077906
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\205\201\335\264"
    dst_address: "\256\027\261\263"
    src_port: 44788
    dst_port: 47652
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200497572
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200497572
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200497572
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "=\022\336\226"
    dst_address: "_\037\222x"
    src_port: 15208
    dst_port: 30515
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200414810
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200414810
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 60
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200414810
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\013\242\257\331"
    dst_address: "_\037\222x"
    src_port: 35224
    dst_port: 28189
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200248286
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200248286
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200248286
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\013\242\257\331"
    dst_address: "\205\201\335\264"
    src_port: 28189
    dst_port: 30515
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200912382
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200912382
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200912382
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\013\242\257\331"
    dst_address: "\"Y\275\211"
    src_port: 45216
    dst_port: 35224
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200580334
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200580334
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 80
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200580334
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "=\022\336\226"
    dst_address: "\205\201\335\264"
    src_port: 19883
    dst_port: 34853
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201160668
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563201160668
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563201160668
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\"Y\275\211"
    dst_address: "Z\315\226a"
    src_port: 1519
    dst_port: 19883
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200413810
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200413810
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200413810
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "=\036\242a"
    dst_address: "\205\201\335\264"
    src_port: 25617
    dst_port: 37855
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200331048
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200331048
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 40
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200331048
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "Z\315\226a"
    dst_address: "_\037\222x"
    src_port: 15322
    dst_port: 44788
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 0
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\"Y\275\211"
    dst_address: "\256\027\261\263"
    src_port: 34853
    dst_port: 28189
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200165524
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200165524
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 20
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200165524
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "Z\315\226a"
    dst_address: "\205\201\335\264"
    src_port: 59641
    dst_port: 28189
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200745858
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200745858
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 100
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200745858
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\"Y\275\211"
    dst_address: "=\036\242a"
    src_port: 44788
    dst_port: 35224
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200995144
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200995144
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 140
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200995144
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\"Y\275\211"
    dst_address: "=\022\336\226"
    src_port: 37855
    dst_port: 15322
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200828620
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200828620
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200828620
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "=\022\336\226"
    dst_address: "\205\201\335\264"
    src_port: 37855
    dst_port: 35224
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200082762
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200082762
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200082762
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "_\037\222x"
    dst_address: "\256\027\261\263"
    src_port: 1519
    dst_port: 37855
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200829620
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200829620
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 120
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200829620
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "=\036\242a"
    dst_address: "_\037\222x"
    src_port: 44788
    dst_port: 34853
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200663096
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200663096
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200663096
    sensor_id: "127.0.0.1:5675"
  }
}
