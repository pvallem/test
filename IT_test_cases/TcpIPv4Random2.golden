flow_info {
  key {
    src_address: "_\037\222x"
    dst_address: "Z\315\226a"
    src_port: 44788
    dst_port: 15322
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 0
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\205\201\335\264"
    dst_address: "=\022\336\226"
    src_port: 35224
    dst_port: 37855
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200082762
    key_type: IPV4
  }
  start_time: 1388563200082762
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200082762
}
flow_info {
  key {
    src_address: "\"Y\275\211"
    dst_address: "\256\027\261\263"
    src_port: 34853
    dst_port: 28189
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200165524
    key_type: IPV4
  }
  start_time: 1388563200165524
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 20
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200165524
}
flow_info {
  key {
    src_address: "_\037\222x"
    dst_address: "\013\242\257\331"
    src_port: 28189
    dst_port: 35224
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200248286
    key_type: IPV4
  }
  start_time: 1388563200248286
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200248286
}
flow_info {
  key {
    src_address: "\205\201\335\264"
    dst_address: "=\036\242a"
    src_port: 37855
    dst_port: 25617
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200331048
    key_type: IPV4
  }
  start_time: 1388563200331048
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 40
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200331048
}
flow_info {
  key {
    src_address: "\"Y\275\211"
    dst_address: "Z\315\226a"
    src_port: 1519
    dst_port: 19883
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200413810
    key_type: IPV4
  }
  start_time: 1388563200413810
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200413810
}
flow_info {
  key {
    src_address: "_\037\222x"
    dst_address: "=\022\336\226"
    src_port: 30515
    dst_port: 15208
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200414810
    key_type: IPV4
  }
  start_time: 1388563200414810
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 60
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200414810
}
flow_info {
  key {
    src_address: "\205\201\335\264"
    dst_address: "\256\027\261\263"
    src_port: 44788
    dst_port: 47652
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200497572
    key_type: IPV4
  }
  start_time: 1388563200497572
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200497572
}
flow_info {
  key {
    src_address: "\"Y\275\211"
    dst_address: "\013\242\257\331"
    src_port: 35224
    dst_port: 45216
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200580334
    key_type: IPV4
  }
  start_time: 1388563200580334
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 80
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200580334
}
flow_info {
  key {
    src_address: "_\037\222x"
    dst_address: "=\036\242a"
    src_port: 34853
    dst_port: 44788
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200663096
    key_type: IPV4
  }
  start_time: 1388563200663096
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200663096
}
flow_info {
  key {
    src_address: "\205\201\335\264"
    dst_address: "Z\315\226a"
    src_port: 28189
    dst_port: 59641
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200745858
    key_type: IPV4
  }
  start_time: 1388563200745858
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 100
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200745858
}
flow_info {
  key {
    src_address: "\"Y\275\211"
    dst_address: "=\022\336\226"
    src_port: 37855
    dst_port: 15322
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200828620
    key_type: IPV4
  }
  start_time: 1388563200828620
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200828620
}
flow_info {
  key {
    src_address: "_\037\222x"
    dst_address: "\256\027\261\263"
    src_port: 1519
    dst_port: 37855
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200829620
    key_type: IPV4
  }
  start_time: 1388563200829620
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 120
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200829620
}
flow_info {
  key {
    src_address: "\205\201\335\264"
    dst_address: "\013\242\257\331"
    src_port: 30515
    dst_port: 28189
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200912382
    key_type: IPV4
  }
  start_time: 1388563200912382
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200912382
}
flow_info {
  key {
    src_address: "\"Y\275\211"
    dst_address: "=\036\242a"
    src_port: 44788
    dst_port: 35224
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200995144
    key_type: IPV4
  }
  start_time: 1388563200995144
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 140
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200995144
}
flow_info {
  key {
    src_address: "_\037\222x"
    dst_address: "Z\315\226a"
    src_port: 35224
    dst_port: 25617
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201077906
    key_type: IPV4
  }
  start_time: 1388563201077906
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 150
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201077906
}
flow_info {
  key {
    src_address: "\205\201\335\264"
    dst_address: "=\022\336\226"
    src_port: 34853
    dst_port: 19883
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201160668
    key_type: IPV4
  }
  start_time: 1388563201160668
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201160668
}
flow_info {
  key {
    src_address: "\"Y\275\211"
    dst_address: "\256\027\261\263"
    src_port: 28189
    dst_port: 15208
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563201243430
    key_type: IPV4
  }
  start_time: 1388563201243430
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 170
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563201243430
}
